pkg/cmd/authTokens.go

```diff
@@ -76,3 +77,21 @@ func (cmd *AuthTokensCommand) createAuthTokensCobraCmd(
 
 	return authTokensCmd, err
 }
+
+func addLoginIdFlagToAuthTokensGet(cmd *cobra.Command, isMandatory bool, authTokensGetCmdValues *AuthTokensCmdValues) {
+
+	flagName := "id"
+	var description string
+	if isMandatory {
+		description = "A mandatory flag that is required to return the access tokens of the currently logged in user."
+	} else {
+		description = "An optional flag that is required to return the access tokens of the currently logged in user."
+	}
+	description += "The input must be a string"
+
+	cmd.PersistentFlags().StringVarP(&authTokensGetCmdValues.name, flagName, "i", "", description)
```

Comment:
PersistentFlags() adds this flag for all sub-commands, which could lead to unwanted behaviour - Flags() should be used